Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ocaml/opam2
+RUN sudo apt install -y m4 perl libgmp-dev pkg-config
+COPY  --chown=opam:opam  .  index 
```

Comment:
you might want to just copy `*.opam` instead of `.`, that will cache the dependencies better.